day1OfJS/variable.js

```diff
@@ -0,0 +1,65 @@
+/*
+Create a variable.js file and declare variables and assign string, boolean, undefined and null data types
+
+let vr = "javascript";
+
+let tr = true;
+let fl = false; 
+
+let firstName
+console.log(firstName);
+
+let emptyValue = null;
+*/
+
+
+/*
+Declare four variables without assigning values
+
+let first
+let second
+let third
+let fouth
+*/
+
+
+/*
+Declare four variables with assigned values
+
+let first = 1;
+let second = 2;
+let third = 3;
+let fouth = 4;
```

Comment:
Remember, the variables must be with const unless you’re going to give them another value later

Reply:
Changed variables - a4cfc15